src/app/modules/library/select-all-projects-checkbox/select-all-projects-checkbox.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { MAT_CHECKBOX_DEFAULT_OPTIONS } from '@angular/material/checkbox';
+
+@Component({
+  selector: 'select-all-projects-checkbox',
+  templateUrl: './select-all-projects-checkbox.component.html',
+  providers: [{ provide: MAT_CHECKBOX_DEFAULT_OPTIONS, useValue: { clickAction: 'noop' } }]
+})
+export class SelectAllProjectsCheckboxComponent {
```

Comment:
It looks like this component has nothing to do with Projects anymore. Should we rename it to ```SelectAllItemsCheckboxComponent```?
```suggestion
export class SelectAllItemsCheckboxComponent {
```